workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java

```diff
@@ -130,7 +131,9 @@ public List<ProjectResponseDTO> getProjects() {
 				hm.get(key).add(projectUserRole);
 			}
 			else {
-				hm.put(key, List.of(projectUserRole));
+				List<ProjectUserRole> pur = new ArrayList<>();
+				pur.add(projectUserRole);
+				hm.put(key, pur);
```

Comment:
nit: 
I think these lines can be `hm.put(key, List.of(projectUserRole));`

Reply:
List.of returns immutable collection which will not be able to add item